### PR TITLE
add LiquiX

### PR DESCRIPTION
**Pull request by danetteceola**
twitter Link: https://twitter.com/liqui_x

List of audit links if any: https://github.com/LiquiX-Finance/Private-Pool-Vault/blob/main/SlowMist%20Audit%20Report%20-%20LiquiX.pdf

Website Link: https://liquix.finance/

Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):https://github.com/LiquiX-Finance/Private-Pool-Vault/tree/main/LiquiX%20Logo%20Pack

Current TVL: $5.98m

Chain: Arbitrum, Optimism, Blast,Polygon

Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list): No Coingecko ID yet

Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000): No Coinmarketcap ID yet

Short Description (to be shown on DefiLlama): A protocol built as one-station LP infrastructure solution.LiquiX offers non-custodial,highly efficient and secure LP service to users via automated execution of algorithm strategies on various dapps and chains aggregated by the protocol.

Token address and ticker if any: Not Issued yet

Category (full list at https://defillama.com/categories) *Please choose only one: Liquidity manager

Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):chainlink

forkedFrom (Does your project originate from another project):No

methodology (what is being counted as tvl, how is tvl being calculated):The funds deposited into the vault and those already invested in LP will be counted as the tvl of the protocol.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/liquix exports TVL: 
        
 
 
    arbitrum                  2.43 M
    
    total                    2.43 M

**Comment by llamatester** [comment]
The adapter at projects/liquix exports TVL: 
        
 
 
    arbitrum                  2.44 M
    
    total                    2.44 M

**Comment by waynebruce0x** [comment]
Can we read vault token balances instead of using totalUnderlying()?

File: projects/liquix/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  doublecounted: true,
+  misrepresentedTokens: true,
+  timetravel: false,
+};
+
+async function getTokens(api, owners) {
+  let tokens = []
+  for (let i = 0; i < owners.length; i++) {
+    const owner = owners[i];
+    let new_tokens = await covalentGetTokens(owner, api, { onlyWhitelisted: true, });
```
Comment (g1nt0ki) [inline projects/liquix/index.js:28]: is there a better way to get token list? `covalentGetTokens` is expensive and we like to keep its usage to a minium

Comment (g1nt0ki) [reply projects/liquix/index.js:28]: nevermind, this method is not even used

**Pull request closed without merging**